concept/thing/statistics.rs

```diff
@@ -933,7 +934,8 @@ mod serialise {
                         .collect();
                     Ok(Statistics {
                         encoding_version: statistics_version,
-                        sequence_number: open_sequence_number,
+                        sequence_number,
+                        last_durable_write_sequence_number: sequence_number,
```

Comment:
Assuming `last_durable_write_sequence_number` would be the one we deserialize from, so that encoding doesn't have to change.